server_release_template/web_gui/js/app/nls/fr/wizard.js

```diff
@@ -0,0 +1,31 @@
+define({
+    "title": "Bienvenue sur ALVR",
+    "subtitle": "Cet assistant va vous aider pour la configuration de base d'ALVR",
+    "titleHardwareReq": "Matériel requis",
+    "textHardwareReq": "ALVR nécésite une carte graphique dédié et recente (Intel HD graphics non supporté). Voici une <a target='_blank' href='https://github.com/polygraphene/ALVR/wiki/Supported-GPU'>liste de GPU supporté</a>. <br/> <br/> Assurez vous d'avoir au moins une sortie audio.",
+    "titleSoftwareReq": "Logiciels requis",
+    "textSoftwareReq": "Assurez vous d'avoir installé SteamVR, et de l'avoir lancé au moins une fois. <br/> <br/> Si vous ne l'avez pas deja fait, veuillez installer la derniere version de <a  target='_blank' href='https://support.microsoft.com/it-it/help/2977003/the-latest-supported-visual-c-downloads'>Visual C++ Redistrubutable x64</a>. <br/> <br/> Pour transemettre l'audio du microphone veuillez installer <a  target='_blank' href='https://www.vb-audio.com/Cable/'>Cable audio virtuel</a>.",
```

Comment:
The Visual C++ package link points to the italian site.

Reply:
fixed

Reply:
thanks!